test/latest/10-canonicalize.js

```diff
@@ -5,14 +5,19 @@ const {expect} = require('chai');
 
 describe('Canonicalize', function() {
   let generatorOptions = null;
-  before(function() {
+  beforeEach(function() {
     generatorOptions = {
       generator: config.generator,
       command: 'c14n',
       args: {},
       date: new Date().toGMTString(),
     };
   });
+  describe('Optional', function() {
```

Comment:
Should this be skipped or a todo?

Reply:
I believe this was addressed in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/b7a8b329f27a3681eaa4c579444fe1cd85ac45b6